olive/model/handler/mixin/dummy_inputs.py

```diff
@@ -59,7 +47,7 @@ def get_dummy_inputs(self, filter_hook=None, filter_hook_kwargs=None):
             dummy_inputs = user_module_loader.call_object(self.dummy_inputs_func, self)
             # respect user's dummy_inputs_func, no hook
         else:
-            dataloader = self.get_dummy_dataloader_from_io_config() or self.get_dummy_dataloader_from_hf_config()
+            dataloader = self._get_dummy_dataloader_from_io_config()
             if dataloader:
                 dummy_inputs, _ = dataloader.get_first_batch()
             elif not self.hf_config.components:
```

Comment:
This is not caused by your PR but could you update this condition to `elif self.hf_config and not self.hf_config.components and self.hf_config.task`? This PR https://github.com/microsoft/Olive/pull/1121 had not added the full conditions for this case.

Reply:
Done!